src/app/components/location-link/location-link.component.ts

```diff
@@ -35,15 +35,19 @@ import { ContentApiService } from '../../services/content-api.service';
 @Component({
     selector: 'aca-location-link',
     template: `
-        <a href="" [title]="tooltip | async" (click)="goToLocation()">
+        <a href="" [title]="nodeLocation$ | async" (click)="goToLocation()">
```

Comment:
@pionnegru I see that the e2e tests are failing and maybe it would be good not to use the async pipe here since this value is calculated only once.